### DISCUSSION TITLE
2007/05/18/%E5%9C%A8%E6%96%87%E4%BB%B6%E5%A4%B9%E4%B8%8B%E7%9B%B4%E6%8E%A5%E5%88%9B%E5%BB%BA%E6%96%B0%E7%9A%84-vi

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#160] [Announcements]
# 2007/05/18/%E5%9C%A8%E6%96%87%E4%BB%B6%E5%A4%B9%E4%B8%8B%E7%9B%B4%E6%8E%A5%E5%88%9B%E5%BB%BA%E6%96%B0%E7%9A%84-vi

在 Windows Explore 中，鼠标右键点击某一文件夹的空白处，弹出的菜单中有“新建”一项。通过修改 Windows 的注册表，可以给这个新建列表添加一项，从而直接在文件夹下创建一个新的 VI。如图1所示。

https://blog.qizhen.xyz/2007/05/18/%E5%9C%A8%E6%96%87%E4%BB%B6%E5%A4%B9%E4%B8%8B%E7%9B%B4%E6%8E%A5%E5%88%9B%E5%BB%BA%E6%96%B0%E7%9A%84-vi

**Comment by ruanqizhen** [comment]
之前博客下的留言：


军
2008年10月10日 15:48 编辑
好呀 可惜压缩包损坏无法打开！

回复
刘伟印
2008年08月15日 13:29 编辑
比较有意思，不过感觉实用性不大，即使快捷菜单创建了New VI要打开还是要启动LabVIEW环境啊

回复
竹贤
2008年01月12日 18:05 编辑
直接修改注册表也可，怎么改？

回复
竹贤
2008年01月11日 11:19 编辑
我觉得有用：有时需要搞一个新项目或新程序，省去了先打开VI环境，简化了操作。这与在桌面新建office的空白文档的功能是一样的，给使用者带来了便利。

回复
Qizhen
2007年05月31日 08:33 编辑
自己造个空的 7.1版 VI，用压缩包里的 VI 转换成注册表数据就可以了。

回复
雨夜
2007年05月29日 19:47 编辑
请问具体是怎样实现的，我自己也想做一个，我用的是lv7.1你的8.0我用不了！

回复
杰
2007年05月20日 22:36 编辑
这个好像不是太有用，嘿嘿。
干活的时候，总是在LV环境中的，如果需要特定的VI，有模版的话，打开这个VI，然后SAVE AS。
要在资源管理器里面直接新建的话，第一没有必要，因为程序总是要自己写的。没有LV环境，建新VI有啥用啊？第二，自己改下注册表就OK。
呵呵，随便说说。

回复
